docs/consumer/Receiver.md

```diff
@@ -1,6 +1,81 @@
 # Stake Receiver
 
 The Stake Receiver is on the consumer side and connected an External Staker on the Provider side.
-This handles the normalization of the token streams and communicates with the Meta-Staker.
+This handles the normalization of the token streams and communicates with "Virtual Staking".
+It is connected to the Provider chain via IBC and "receives" the various staking commands
+
+**TODO** This was called the "Consumer contract" in previous versions and probably needs a better name
```

Comment:
`mesh-consumer` isn't such a bad name IMO, it implies a relationship with a `mesh-provider`... glad we got rid of `meta-staking` but maybe the current consumer / provider paradigm isn't so bad?

Reply:
Though, `mesh-provider` / `mesh-receiver` isn't so bad actually.